src/scraper/fwd.h

```diff
@@ -53,13 +53,33 @@ typedef std::map<std::string, CSerializeData> mConvergedManifestParts;
 struct ConvergedManifest
 {
     // IMPORTANT... nContentHash is NOT the hash of part hashes in the order of vParts unlike CScraper::manifest.
-    // It is the hash of the data in the ConvergedManifestPartsMap in the order of the key.
+    // It is the hash of the data in the ConvergedManifestPartsMap in the order of the key. It represents
+    // the composite convergence by taking parts piecewise in the case of the fallback to bByParts (project) level.
     uint256 nContentHash;
     uint256 ConsensusBlock;
     int64_t timestamp;
     bool bByParts;
 
     mConvergedManifestParts ConvergedManifestPartsMap;
+
+    // Used when convergence is at the manifest level (normal)
+    std::map<ScraperID, uint256> mIncludedScraperManifests;
```

Comment:
Do we have plans to use this field for something in the future? It looks like it could be a local variable in `ScraperConstructConvergedManifest()`.

Reply:
There are several maps in the struct that we are not using, but I have left them in there because I am not sure quite yet what all I am going to put in the scraper status function. There are also two multimaps that are inverses of one another to deal with the many to many situation that actually exists in the fallback by project scenario.  I will weed the unused ones out once I have written the rpc function scraperstatus we were talking about in Slack earlier.